parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -686,11 +689,14 @@ impl snowbridge_control::Config for Runtime {
 	type MessageHasher = BlakeTwo256;
 	type WeightInfo = weights::snowbridge_control::WeightInfo<Runtime>;
 	type MaxUpgradeDataSize = MaxUpgradeDataSize;
-	type AgentHashedDescription = AgentHashedDescription;
+	type AgentIdOf = AgentIdOf;
 	type UniversalLocation = UniversalLocation;
 	type RelayLocation = RelayLocation;
 	type AgentOrigin = EnsureXcm<Everything>;
 	type ChannelOrigin = EnsureXcm<AllowSiblingsOnly>;
+	type LocalPalletId = TreasuryPalletId;
```

Comment:
Use Treasury pallet id.

Reply:
Shouldn't we name this type `TreasuryPalletId` for clarity?